examples/bpk-component-modal-v2/examples.tsx

```diff
@@ -91,8 +91,13 @@ const content = [
   </Paragraph>,
 ];
 
-const ModalContainer = (props: Omit<BpkModalProps, "id" | "isOpen" | "ariaLabelledby" | "closeLabel" | "onClose">) => {
-  const [isOpen, setOpen] = useState(false);
+const ModalContainer = (
+  props: Omit<
+    BpkModalProps,
+    'id' | 'ariaLabelledby' | 'closeLabel' | 'onClose'
+  >,
+) => {
+  const [isOpen, setOpen] = useState(props.isOpen || false);
```

Comment:
If we are using the prop in this way we should be clear with the naming, something like `props.initiallyOpen` would be more suitable.